packages/quantic/force-app/main/default/lwc/quanticModal/quanticModal.css

```diff
@@ -0,0 +1,60 @@
+.modal {
+  background-color: white;
```

Comment:
If possible, I think we should use the `$color-background-alt` design token.
See [Design Tokens](https://www.lightningdesignsystem.com/design-tokens/#category-background-color)